test/unit_tests/apps/financial-well-being/fwb-questions.js

```diff
@@ -186,8 +190,8 @@ describe( 'fwb-questions', () => {
       expect( formSubmissionStatus ).to.equal( false );
     } );
 
-  it( 'submit button should submit the form ' +
-      'if all the questions are completed before page load.',
+  xit( 'submit button should submit the form ' +
```

Comment:
These aren't testing form submission; they are testing if `event.preventDefault` was [called](https://github.com/cfpb/cfgov-refresh/blob/5b9987de360a49089b54f1b6fc1cd528f0368a02/cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js#L55) when the click event was fired. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent.

I think these test are fine.

Reply:
They won't pass as is because they trigger a form submission in the DOM, but yeah could be re-written to not.

Reply:
yeah, I would just add in the before statement:

```
    document.querySelector( '#quiz-form' ).addEventListener( 'submit', event => event.preventDefault() )'
```

Reply:
Refactored the test, try it now

Reply:
For some reason it ignored the submission event in the before, but works in the trigger click event.